package.json

```diff
@@ -7,9 +7,10 @@
         "test": "mocha --expose-gc --timeout 40000",
         "start": "node ../server.js",
         "lint": "eslint .",
-        "test-on-github": "mocha --expose-gc --timeout 40000 --ignore test/test_filesFromWild.js"
+        "test-on-github": "mocha --expose-gc --timeout 40000 --ignore test/test_filesFromWild.js --ignore test/test_cli.js"
     },
     "dependencies": {
+        "concat-stream": "^2.0.0",
```

Comment:
Should this be a dev-dependency?

Reply:
yes. It is used in testing. So we can move it to dev-dependencies. Shall change that

Reply:
Yes, let's do that.

Reply:
done